modules/core/src/webgl/classes/texture-cube.js

```diff
@@ -137,12 +137,14 @@ export default class TextureCube extends Texture {
     return this;
   }
 
-  bind({index} = {}) {
-    if (index !== undefined) {
-      this.gl.activeTexture(GL.TEXTURE0 + index);
+  bind(textureUnit = this.textureUnit) {
```

Comment:
Is this code identical to `texture-2d.js`? Can it be shared in `texture.js`?

Reply:
No, is not the same. The other one throws error when index is not there.

Reply:
Does it need to be different? Does the other one need to throw an error. Do we want to track these issues so that we can make an alignment pass later, maybe in a tracker task?